graph/interfaces.go

```diff
@@ -147,8 +147,8 @@ type DB interface {
 	// ChanUpdatesInHorizon returns all the known channel edges which have
 	// at least one edge that has an update timestamp within the specified
 	// horizon.
-	ChanUpdatesInHorizon(startTime, endTime time.Time) (
-		[]channeldb.ChannelEdge, error)
+	ChanUpdatesInHorizon(startTime, endTime time.Time, startBlock,
```

Comment:
Is the expectation that both the timestamp _and_ block are always set now? If only one of them is expecting, we may wan tto rol that into a new struct to better convey that with types.